src/api/EscargotPublic.h

```diff
@@ -1455,6 +1455,9 @@ class ESCARGOT_EXPORT FunctionObjectRef : public ObjectRef {
 
     bool isConstructor();
     void markFunctionNeedsSlowVirtualIdentifierOperation();
+
+    // set function name property
+    void setName(AtomicStringRef* name);
```

Comment:
How about add comment about what function is allowed to changed named property?
and we can return bool value for the operation was successful

Reply:
Good idea! I fixed it